test/e2e/suite_test.go

```diff
@@ -173,6 +176,11 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	framework.InitClusterInformation(karmadaClient, controlPlaneClient)
 
+	controllerManager = framework.InitControllerManagerAndStartCache(restConfig)
+
+	err = utilhelper.IndexWork(context.TODO(), controllerManager)
+	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
 	testNamespace = fmt.Sprintf("karmadatest-%s", rand.String(RandomStrLength))
 	err = setupTestNamespace(testNamespace, kubeClient)
```

Comment:
Hi, @RainbowMango 

I just noticed this part, is it all e2e tests are using a shared cluster?

Reply:
Just checked ci and scripts and it seems the cluster is built via local-up-karmada.sh, so I have no questions now.